### PR TITLE
Add some highlights to changelog

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,25 @@
 
 ## Next release
 
-...
+## `--strict-bytes`
+
+By default, mypy treats an annotation of ``bytes`` as permitting ``bytearray`` and ``memoryview``.
+[PEP 688](https://peps.python.org/pep-0688) specified the removal of this special case.
+Use this flag to disable this behavior. `--strict-bytes` will be enabled by default in *mypy 2.0*.
```
Comment (cdce8p) [inline CHANGELOG.md:9]:
```suggestion
Use this flag to disable this behavior. `--strict-bytes` will be enabled by default in **mypy 2.0**.
```
Should it be bold instead?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,25 @@
...
+## Improvements to partial type handling in loops
+
+This change results in mypy better modelling control flow within loops and hence detecting several
+issues it previously did not detect. In some cases, this change may require use of an additional
+explicit annotation of a variable.
+
+Contributed by Christoph Tyralla (PR [18180](https://github.com/python/mypy/pull/18180)).
+
+(Speaking of partial types, another reminder that mypy plans on enabling `--local-partial-types`
+by default in *mypy 2.0*).
```
Comment (cdce8p) [inline CHANGELOG.md:23]:
```suggestion
by default in **mypy 2.0**).
```
Same here.

**Review by cdce8p — APPROVED** [review]
Looks good! Left a small suggestion. We can always refine it later when writing the final release notes for `1.15`.

_Got me thinking, I could also add a section for dropping Python 3.8._

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,25 @@
 
 ## Next release
 
-...
+## `--strict-bytes`
```
Comment (cdce8p) [inline CHANGELOG.md:5]:
```suggestion
### `--strict-bytes`
```
This needs to be indented one more level.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,25 @@
...
+
+By default, mypy treats an annotation of ``bytes`` as permitting ``bytearray`` and ``memoryview``.
+[PEP 688](https://peps.python.org/pep-0688) specified the removal of this special case.
+Use this flag to disable this behavior. `--strict-bytes` will be enabled by default in *mypy 2.0*.
+
+Contributed by Ali Hamdan (PR [18137](https://github.com/python/mypy/pull/18263/)) and
+Shantanu Jain (PR [13952](https://github.com/python/mypy/pull/13952)).
+
+## Improvements to partial type handling in loops
```
Comment (cdce8p) [inline CHANGELOG.md:14]:
```suggestion
### Improvements to partial type handling in loops
```

**Review by cdce8p — CHANGES_REQUESTED** [review]

**Comment by hauntsaninja** [comment]
Thanks!

**Review by cdce8p — APPROVED** [review]

**Pull request merged**